tests/jsTests/errorSpec.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Suite : Tests for showing error on incorrect field entry
+ * Description : This is a test suite for showing error in case of an invalid field entry
+ */
+describe("Tests for showing error on incorrect field entry", function() {
+	var msg = 'This is a test message for showing error';
+	beforeEach(function() {
+		setFixtures('<div class="field"> \
+				<div class="ui red pointing"></div> \
+			</div>');
+		showError($('.field'), msg);
+	});
+
+	it("should be visible", function() {
+		expect($('.field')).toHaveClass('error');
+		expect($('.ui.red.pointing')).toContainText(msg);
+		expect($('.ui.red.pointing')).toBeVisible();
+	});
+});
+
+/**
+ * Suite : Tests for hiding error on correct field entry
+ * Description : This is a test suite for hiding error in case of an valid field entry
+ */
+describe("Tests for hiding error on correct field entry", function() {
+	var msg = 'This is a test message for not showing error';
+	beforeEach(function() {
+		setFixtures('<div class="field"> \
+				<div class="ui red pointing">' + msg + '</div> \
+			</div>');
+		hideError($('.field'));
```

Comment:
@fatimarafiqui 
https://github.com/systers/FirstAide-web/pull/264/files/68637a7ce80b3d5487de124ba90dad59ff402280#diff-5225fdc1ab35e654deeac6f8b75ab2acR24 Thats the hideError method that you have written. The method removes 'error' class. In your setFixtures, you need to set the 'error' class, then call `hideError` and then add the assert to see if the `'error'` class is removed from the html.  In this case, the 'error' class is not present even before calling the hideError method, so it true by default.